### PR TITLE
Fix#71 : Login Fragment fix

### PR DESCRIPTION
**Pull request by codedsun**
Fix#71: Changes in onBackPressed( )

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt [hunk 1]
```diff
@@ -48,4 +48,8 @@ class AuthActivity : AppCompatActivity() {
                 .replace(R.id.frame_container_auth, fragment)
                 .commit()
     }
+
+    override fun onBackPressed() {
+        loadFragment(ProfileFragment())
+    }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt]: No, this will start a whole new back stack

Comment (codedsun) [reply app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt]: @iamareebjamal Then either we have to start activity with the flag of clear_top, else we are having root view shown of the fragment in the backstack

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt]: Make this a fragment instead

Comment (codedsun) [reply app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt]: @iamareebjamal Okay sure!

**Comment by codedsun** [comment]
@iamareebjamal I am not able to push changes, I dont' know 
`codedsun wants to merge 1 commit into fossasia:master from unknown repository`

**Comment by codedsun** [comment]
Forwarding to PR #85 ! Closing this PR

**Pull request closed without merging**